packages/gatsby/src/internal-plugins/query-runner/__tests__/file-parser.js

```diff
@@ -81,7 +123,11 @@ export const fragment3 = graphql\`
   })
 
   it(`extracts query AST correctly from files`, async () => {
+    let outputData = ``
+    let storeLog = inputs => (outputData += inputs)
+    console[`log`] = jest.fn(storeLog)
```

Comment:
this doesn't seem to catch `report.warn` (I think you changed this warning message from `console.log` to `report.warn` yesterday?

Reply:
Yeah it is — https://github.com/gatsbyjs/gatsby/pull/7801/files#diff-ebd8e6506267979cbcf1d94405030881R860

`report.warn` uses console.log under the hood.

Reply:
oh, the test is failing now 😅 yeah, you're right. Investigating...